aws-cloud-integrations.md

```diff
@@ -328,15 +328,17 @@ Download template files from the URLs provided below and upload them as the stac
 ### Step 4: Attaching IAM permissions to Kubecost
 Now that the policies have been created, attach those policies to Kubecost. We support the following methods:
 
+> **NOTE:** If you are using the alternative [multi-cloud integration](https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/multi-cloud) method steps 4 and 5 are not required. The use of "Attach via Pod Annotation on EKS" authentication is optional.
+
 <details>
 	<summary>Attach via ServiceKey And Kubernetes Secret</summary>
 <ul>
 <li><p>Navigate to <a href="https://console.aws.amazon.com/iam">https://console.aws.amazon.com/iam</a> Access Management &gt; Users. Find the Kubecost User and select Security Credentials &gt; Create Access Key. Note the Access key ID and Secret access key. You&rsquo;ll use it to either Create a secret from helm values or Create and use an existing secret.</p>
 	<details>
 		<summary>Create a secret from Helm values</summary>
 
-<p><details>
-	<summary>Create a secret from Helm values</summary></p>
+<details>
```

Comment:
When Github renders this Markdown, I see two dropdowns that say "Create a secret from Helm values". Was there old text that got deleted?

Reply:
That "Create a secret from Helm values" section is not showing up on the gitbooks page, this was an attempt to fix that. 

I could have made it worse.

https://docs.kubecost.com/install-and-configure/advanced-configuration/cloud-integration/aws-cloud-integrations#step-4-attaching-iam-permissions-to-kubecost

Reply:
Yep, that made it worse in GH... It appears on the docs page the sub bullets are not rendering at all. I will revert this change here. Seems like we may need a larger check of the whole docs site to see is issues are elsewhere.

Reply:
I reverted it back, but it still shows twice because of how it was orriginally.